lib/ex_debug_toolbar/request/event.ex

```diff
@@ -0,0 +1,10 @@
+defmodule ExDebugToolbar.Request.Event do
+  defstruct [
+    name: nil,
+    started_at: nil,
+    finished_at: nil,
+    duration: 0,
+    metadata: %{},
```

Comment:
with metadata and events, we are getting ahead of ourselves. We are not sure what are wee going to track there